src/utils/error_checking.hpp

```diff
@@ -51,19 +51,16 @@ namespace ErrorChecking {
 KOKKOS_INLINE_FUNCTION
 void require(std::string const &condition, std::string const &message,
              std::string const &filename, int const linenumber) {
-  fprintf(stderr,
-          "### PARTHENON ERROR\n  Condition:   %s\n  Message:     %s\n  File:        "
-          "%s\n  Line number: %i\n",
-          condition.c_str(), message.c_str(), filename.c_str(), linenumber);
+  printf("### PARTHENON ERROR\n  Condition:   %s\n  Message:     %s\n  File:        "
+         "%s\n  Line number: %i\n",
+         condition.c_str(), message.c_str(), filename.c_str(), linenumber);
   exit(EXIT_FAILURE);
```

Comment:
Presumably the MR that fixes this should go in before this one.

Reply:
Yes.

Reply:
make sure to merge master in, since this has been fixed in master now.